mappings/com/mojang/realmsclient/client/Errable.mapping

```diff
@@ -1,3 +1,3 @@
-CLASS net/minecraft/class_5221
+CLASS net/minecraft/class_5221 com/mojang/realmsclient/client/Errable
```

Comment:
Not sure that's what that actually is. I have ErrorReceiver in #1353 which is also bad though.

Reply:
`Errable` doesn't look like anything to me

Reply:
Errable means able to error.